customcommands/web.go

```diff
@@ -643,11 +644,16 @@ func handleRunCommandNow(w http.ResponseWriter, r *http.Request) (web.TemplateDa
 	}
 
 	// only interval commands can be ran from the dashboard currently
-	cmd, err := models.CustomCommands(qm.Where("guild_id = ? AND local_id = ? AND trigger_type = 5", activeGuild.ID, cmdID)).OneG(context.Background())
+	cmd, err := models.CustomCommands(qm.Where("guild_id = ? AND local_id = ? AND trigger_type = 5", activeGuild.ID, cmdID), qm.Load("Group")).OneG(context.Background())
 	if err != nil {
 		return templateData, err
 	}
 
+	if cmd.GroupID.Int64 != 0 && cmd.R.Group.Disabled {
```

Comment:
(untested)
```suggestion
	if cmd.R.Group != nil && cmd.R.Group.Disabled {
```
is probably less brittle and ditto everywhere else.